designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/NavigationBar.kt

```diff
@@ -0,0 +1,110 @@
+package org.hisp.dhis.mobile.ui.designsystem.component
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.offset
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.NavigationBarItemColors
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import org.hisp.dhis.mobile.ui.designsystem.theme.Outline
+import org.hisp.dhis.mobile.ui.designsystem.theme.SurfaceColor
+import org.hisp.dhis.mobile.ui.designsystem.theme.TextColor
+
+@Composable
+fun NavigationBar(
+    modifier: Modifier = Modifier,
+    items: List<NavigationBarItem>,
+    selectedItemIndex: Int? = null,
+    onItemClick: (Int) -> Unit,
+) {
+    Column(
+        modifier = modifier.testTag("NAVIGATION_BAR_CONTAINER"),
```

Comment:
What do you think about extracting the tags to constant in order to use it for testing instead of hardcoding?

Reply:
Fixed